### ISSUE TITLE
Different serialisation of "supersedes"

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-nist#66]
NIST differentiates between the relations `supersedes` and `obsoletes`. It turns out that `obsoletes` is the only relation type supported by the Relaton grammar. A relation of type `supersedes` needs to be expressed as a specialisation of `obsoletes`. It needs to be serialised into XML as

````
<relation type="obsoletes"><description>supersedes</description>
<bibitem> ... </bibitem>
</relation>
````

**Comment by andrew2net** [comment]
@opoudjis but the [NIST grammar](https://github.com/metanorma/metanorma-model-iso/blob/c25f58187774246aebc5771631c0985da3db07bf/grammars/nist.rnc#L35) supports `supersedes` relation.

**Comment by andrew2net** [comment]
fixed in v 1.9.8
